src/Core/View.js

```diff
@@ -72,7 +72,7 @@ function _preprocessLayer(view, layer, parentLayer) {
     if (layer.isLabelLayer) {
         view.mainLoop.gfxEngine.label2dRenderer.registerLayer(layer);
     } else if (layer.labelEnabled) {
-        const labelLayer = new LabelLayer(`${layer.id}-label`, view.referenceCrs);
+        const labelLayer = new LabelLayer(`${layer.id}-label`, { crs: view.referenceCrs });
```

Comment:
`crs` will be always `view.referenceCrs`?

Reply:
Yes, I change things in `LabelLayer` and removed that